target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -45,6 +45,15 @@ nbg6817)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "$board:amber:wifi5" "phy0radio"
 	ucidef_set_led_netdev "wan" "WAN" "$board:white:internet" "eth1"
 	;;
+vr2600v)
+	ucidef_set_led_default "dsl" "DSL" "${board}:white:dsl" "0"
+	ucidef_set_led_usbport "usb" "USB" "${board}:white:usb" "usb4-port1" "usb2-port1"
```

Comment:
I tested it with a USB3 device and it was working. There are only 2 USB ports on this device. Maybe this will not work for USB2 devices?